.husky/pre-push

```diff
@@ -2,3 +2,4 @@
 . "$(dirname -- "$0")/_/husky.sh"
 
 yarn lint
+yarn build
```

Comment:
I think we just need "yarn test". vercel will do lint and build automatically.

Reply:
however, yarn test should only be run locally and not in staging (aka development) or production environments, because it modifies the database.